app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -118,27 +154,61 @@ function useGrantDetail() {
   });
   const grantMetadata = computed(() => (grant.value ? metadata.value[grant.value.metaPtr] : null));
 
+  // --- Check properties ---
+  const hasWebsite = computed(() =>
+    grant.value
+      ? metadata.value[grant.value.metaPtr]?.properties?.projectWebsite !== undefined &&
+        metadata.value[grant.value.metaPtr]?.properties?.projectWebsite !== ''
+      : false
+  );
+  const hasGithub = computed(() =>
+    grant.value
+      ? metadata.value[grant.value.metaPtr]?.properties?.projectGithub !== undefined &&
+        metadata.value[grant.value.metaPtr]?.properties?.projectGithub !== ''
+      : false
+  );
+  const hasHandle = computed(() =>
+    grant.value
+      ? metadata.value[grant.value.metaPtr]?.properties?.twitterHandle !== undefined &&
+        metadata.value[grant.value.metaPtr]?.properties?.twitterHandle !== ''
+      : false
+  );
```

Comment:
These all follow the same structure so you might be able to make things a bit more DRY with a helper method that takes the field name, e.g. `projectWebsite` as an input. Maybe that's clunky/harder to read though. I'm fine with either